pkg/apis/jvmbuildservice/v1alpha1/artifactbuildrequest_types.go

```diff
@@ -27,8 +26,6 @@ const (
 	ArtifactBuildRequestStateNew = "ArtifactBuildRequestNew"
 	// ArtifactBuildRequestStateDiscovering The discovery pipeline is running to try and figure out how to build this artifact
 	ArtifactBuildRequestStateDiscovering = "ArtifactBuildRequestDiscovering"
-	// ArtifactBuildRequestStateDiscovered The discovery process is complete
-	ArtifactBuildRequestStateDiscovered = "ArtifactBuildRequestDiscovered"
```

Comment:
ok this lines up with what i recall from my avoid multiple status writers POC, so I take this as confirming / good